esnext-browsers.js

```diff
@@ -284,6 +284,18 @@ module.exports = {
       short: 'CH 52',
       note_id: 'experimental-flag',
   },
+  chrome53: {
```

Comment:
This is not what I am suggesting.  What I suggested was simply removing the 53 column while there's no actual need for it.  As I understand, you then suggested also removing 52, and I agree.